### PR TITLE
Added functionality for add new location on builder page

### PR DESCRIPTION
**Pull request by AnastasiiaHamaiunova**
Some styles.

### CODE REVIEW

**Review by RostyslavSt — APPROVED** [review]
+

File: src/reducers/builder.reducer.js [hunk 1]
```diff
@@ -104,6 +104,17 @@ const reducer = (state = initialState, action) => {
         ...state,
         locations: action.locations
       };
+
+    case 'ADD_LOCATION_SUCCESS': {
+      const newLocations = state.locations;
```
Comment (totoshk) [inline src/reducers/builder.reducer.js]: In the variable newLocations, you wrote down the reference to the array. You need to use the spread operator.

File: src/components/DeviceForm/deviceForm.scss [hunk 2]
```diff
@@ -8,6 +8,12 @@
   border-bottom: 1px solid #3d404c;
   color:$text_color_active;
 
+    &__icon{
+      cursor: pointer;
+      color: $light-gray;
+
```
Comment (totoshk) [inline src/components/DeviceForm/deviceForm.scss]: Extra line.

File: src/components/DeviceForm/deviceForm.js [hunk 3]
```diff
@@ -29,14 +30,49 @@ class DeviceForm extends React.Component {
...
+          onClick={this.addLocationValue}>
+        </i>
+      </div>);
+  };
+  changeLocationValue = (e) => {
+    const newLocationValue = e.target.value;
+
+    console.log(newLocationValue);
+    this.setState({
+      ...this.state,
```
Comment (totoshk) [inline src/components/DeviceForm/deviceForm.js]: The spread operator is unnecessary.

**Review by totoshk — APPROVED** [review]
Approve, but you need to take into account the comments.

**Pull request merged**